packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -227,15 +227,12 @@ export class SummarizerNode implements IRootSummarizerNode {
 
             const props = {
                 summaryRefSeq,
-                pendingSize: this.pendingSummaries.size,
-                pendingHandle: this.pendingSummaries.size > 0 ? this.pendingSummaries.keys[0] : undefined,
-                pendingSeqNumber: this.pendingSummaries.size > 0 ?
-                    this.pendingSummaries.values[0].referenceSequenceNumber : undefined,
```

Comment:
This mostly looks fine to me, except `this.pendingSummaries.values[0].referenceSequenceNumber` which looks wrong, but could probably just be changed to ` this.pendingSummaries.values[0]?.referenceSequenceNumber`

Reply:
yes, I believe we should be fine. From all the loggings, there wasn't a single entry in which the size > 0 - so I don't think we need the other fields.